google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -193,7 +193,7 @@ clouddebug.errorset=The snapshot location could not be set.
 clouddebug.continuesession=Continue Session
 clouddebug.continueanyway=Continue Anyway
 clouddebug.text=Google Stackdriver Debug
-clouddebug.stackdriver.debugger=Stackdriver Debugger
+clouddebug.stackdriver.debugger=Google Stackdriver Debugger
```

Comment:
is this still being used now that you changed `getConfigurationTypeDescription`?